src/components/record/RecordForm.tsx

```diff
@@ -76,6 +76,14 @@ export default function RecordForm(props: Props) {
     bucket,
   } = props;
 
+  const {
+    data: { schema = {}, uiSchema = {}, attachment },
+  } = collection;
+  const attachmentConfig = {
+    enabled: attachment?.enabled,
+    required: attachment?.required && !record?.data?.attachment
```

Comment:
Shall we add a comment here making it explicit that this allows records to be edited without attachments to be reuploaded?